### PR TITLE
(blog)Add challenge 5 of 100DaysOfGatsby

### PR DESCRIPTION
**Pull request by hashimwarren**
This prompts invites developers to mix markdown with React using MDX

### CODE REVIEW

File: docs/blog/100days/mdx/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Challenge 5 - Add React Components in Markdown
+date: 2020-01-29
+author: "Hashim Warren"
+excerpt: "Learn how tomMix markdown and React using MDX"
```
Comment (laurieontech) [inline docs/blog/100days/mdx/index.md]:
```suggestion
excerpt: "Learn how to mix markdown and React using MDX"
```

File: docs/blog/100days/mdx/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+_Gatsby was named the [#1 new technology to learn in 2020](https://www.cnbc.com/2019/12/02/10-hottest-tech-skills-that-could-pay-off-most-in-2020-says-new-report.html)!_
+
+_To help you learn about Gatsby and our ecosystem, we invite you to take the #100DaysOfGatsby challenge! If you are new to Gatsby, you can follow along with our beginner-friendly weekly prompts. Or if you are familiar with Gatsby and want to go deeper, we invite you to [build your first Gatsby Theme](/docs/themes/building-themes/)._
+
+_Learn more about [#100DaysOfGatsby here](/blog/100days)!_
+
+## Challenge 5: Mix Markdown and React Using MDX
+
+React component within your Markdown-based blog posts!
```
Comment (laurieontech) [inline docs/blog/100days/mdx/index.md]:
```suggestion
Use React components within your Markdown-based blog posts!
```

File: docs/blog/100days/mdx/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Markdown is a great markup language for creating human readable, long form content. A writer on your team can use Markdown’s friendly syntax and contribute content to a site without having to edit HTML or JSX directly. 
+
+Today we invite you to use MDX, a new format that lets you add components in your Markdown documents. Now, editors can add interactive and styled elements to their documents, not just static, plain text. If you are familiar with using shortcodes in the classic WordPress editor, MDX enables you to have a similar editing workflow.
```
Comment (laurieontech) [inline docs/blog/100days/mdx/index.md]: If you're not familiar with WP shortcodes is there a link we can give you? Or maybe drop this comparison? It feels like it works well for a specific audience, but might be intimidating for others.

Comment (hashimwarren) [reply docs/blog/100days/mdx/index.md]: Removed!

**Pull request merged**